source/mysql/tests/test_source.py

```diff
@@ -67,7 +67,6 @@ def setUpClass(cls):
     @classmethod
     def tearDownClass(cls):
         super().tearDownClass()
-        cls._exit_stack.__exit__(None, None, None)
```

Comment:
Was there a reason this was removed?

Reply:
It would throw this error: `AttributeError: type object 'TestDbSource' has no attribute '_exit_stack'`

I'm not too sure how the _exit_stack stuff works, so I didn't see the use it added.

Reply:
This is in mysql source though?